test/cpp/jit/test_upgrader_utils.cpp

```diff
@@ -0,0 +1,22 @@
+#include <gtest/gtest.h>
+#include <torch/csrc/jit/operator_upgraders/utils.h>
+#include <torch/csrc/jit/operator_upgraders/version_map.h>
+
+#include <test/cpp/jit/test_utils.h>
+
+namespace torch {
+namespace jit {
+
```

Comment:
Could we add test cases for other utilities like `isOpCurrent`?

Reply:
sounds good

Reply:
done